python/mspasspy/db/database.py

```diff
@@ -10,8 +10,12 @@
 import urllib.request
 from array import array
 import pandas as pd
-import dask.bag as daskbag
-import dask.dataframe as daskdf
+
+try:
+    import dask.bag as daskbag
+    import dask.dataframe as daskdf
+except:
+    pass
```

Comment:
This should decide which should be use as the default `format` argument in the `read_distributed_data`. If neither is available, we should just hide `read_distributed_data`.